.github/workflows/workflow_inference.yml

```diff
@@ -107,7 +107,7 @@ jobs:
           if [[ ${{ matrix.model }} == "mpt-7b-bigdl" ]]; then
             docker exec "${TARGET}" bash -c "python inference/serve.py --config_file inference/models/bigdl/mpt-7b-bigdl.yaml --serve_simple"
           else
-            docker exec "${TARGET}" bash -c "MODEL_TO_SERVE=\"${{ matrix.model }}\" python inference/serve.py --serve_simple"
+            docker exec "${TARGET}" bash -c "python inference/serve.py --serve_simple --model_to_serve ${{ matrix.model }}"
```

Comment:
do you also think --serve_simple and --model_to_serve is too verbose since the command is already serve.py?
Does it make sense to change to --simple and --model?

Reply:
I think it's better to keep --serve_simple, how about modifying --model_to_serve to --choosed_models, also gentellly ping @jiafuzha here.

Reply:
I agree with xiaochang to use --simple and --models since serve.py already means 'serve', right?

Reply:
OK, got it.